app/controllers/general.php

```diff
@@ -936,10 +936,10 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
                 ->setParam('development', App::isDevelopment())
                 ->setParam('projectName', $project->getAttribute('name'))
                 ->setParam('projectURL', $project->getAttribute('url'))
-                ->setParam('message', $error->getMessage())
-                ->setParam('type', $type)
-                ->setParam('code', $code)
-                ->setParam('trace', $trace);
+                ->setParam('message', $output['message'] ?? '')
+                ->setParam('type', $output['type'] ?? '')
+                ->setParam('code', $output['code'] ?? '')
```

Comment:
only trace can be null here 👍🏻

Reply:
I added this just in case someone above me messes things up, its usually a best practice when accessing keys in an assoc array.